src/client/components/rich-text/rich-text.vue

```diff
@@ -27,14 +27,15 @@ export default {
 .rich-text blockquote {
   position: relative;
   margin: var(--spacing-double) 0;
-  padding: 1.3rem 0;
+  padding-top: 1.3rem;
+  padding-bottom: 0;
 }
 
 .rich-text blockquote:before {
   content: open-quote;
   position: absolute;
   left: 0;
-  top: -35px;
+  top: -30px;
```

Comment:
Can you make this based on ems (so that you don't have to redeclare them in other media queries (see below)?